src/values/ObjectValue.js

```diff
@@ -91,6 +92,7 @@ export default class ObjectValue extends ConcreteValue {
     "$StringIteratorNextIndex",
     "$WeakMapData",
     "$WeakSetData",
+    "$SuperTypeParameters",
```

Comment:
This should not be tracked:
1. Tracking is not needed, as the property is only set once directly after object creation, and is never mutated again, or along different execution paths. Tracking is all about joining modifications of (pre-existing) object properties when such modifications happen along alternatively explored execution paths during symbolic execution.
2. Tracking is only supported for properties that have type `Value`, or possible a subtype of `Value` plus `AbstractValue`. There's some special support for arrays to support map/set data, but that's a big hack. `typeAnnotations` are certainly not supported.
3. Tracking imposes a substantial overhead which is just not needed here.